lib/modules/dosomething/dosomething_reportback/dosomething_reportback.forms.inc

```diff
@@ -370,6 +370,9 @@ function dosomething_reportback_form_validate_file($form, &$form_state) {
   }
   // If this is a new reportback form, file is mandatory.
   elseif ($form_state['values']['rbid'] == 0) {
+    $ga = new Elements_Tools_Serversideanalytics();
```

Comment:
I'm not sure this class exists without installing some extra packages... we have a [custom helper function](https://github.com/DoSomething/phoenix/blob/44e8094e68ef24dcfe9c4e8875c68960ab52578f/lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module#L527-L545) that you can use to inject a analytics event on the following page load.